.jobserv.yml

```diff
@@ -4,7 +4,7 @@ triggers:
     type: github_pr
     runs:
       - name: unit-test
-        container: golangci/golangci-lint:v1.48
```

Comment:
Probably this commit should be extracted into the separate PR?

Reply:
Probably, you're right.

The CI was failing for me about bad format (while locally the auto-formatter worked well), so I was guessing about the difference between the formatter/linter/golang versions here and there.

Now that I made the `make check` pass in the CI, I could try to extract these non-related commits into a separate PR.